Helper/ShipmentOptions.php

```diff
@@ -113,7 +113,7 @@ public function hasAgeCheck(): bool
             return false;
         }
 
-        $ageCheckFromOptions  = self::getValueOfOptionWhenSet($this->options, 'age_check');
+        $ageCheckFromOptions  = self::getValueOfOptionWhenSet('age_check', $this->options);
```

Comment:
Ik denk dat we voor 'age_check' een constante hebben. Als we die niet hebben dan moet je de even aanmaken en ook op andere plekken gebruiken.